tests/test_vxlan_vtep_vni.rb

```diff
@@ -49,6 +49,8 @@ def test_create_with_existing
   end
 
   def test_vnis
+    skip('Platform does not support vnis') if VxlanVtepVni.vnis['nve1'].nil?
```

Comment:
Drat. I'd forgotten that we added the auto-skip logic to develop as well. If at all possible, you should port #159 (probably just the changes to basetest.rb and test_command_reference.rb) to develop and fix up the resulting failures. Otherwise, we'll be on the hook for fixing this when we do the feature/grpc branch collapse.